addons/website_sale/views/templates.xml

```diff
@@ -74,6 +74,7 @@
             <attribute name="data-display-description">true</attribute>
             <attribute name="data-display-price">true</attribute>
             <attribute name="data-display-image">true</attribute>
+            <attribute name="data-display-rating">true</attribute>
```

Comment:
Should not be true by default since visitors do not have the possibility to rate unless the option is enabled